Svc/PrmDb/test/int/test_cmd_PrmDb.py

```diff
@@ -0,0 +1,91 @@
+""" test_cmd_PrmDb.py:
+
+Test the command dispatcher with basic integration tests.
+"""
+import time
+
+def test_send_PrmDb(fprime_test_api):
+    """Test that commands may be sent
+
+     Tests command send, dispatch, and receipt using send_and_assert command with a pair of CmdDispatcher commands.
+
+     # remove file PrmDB.dat
+
+     recvBuffComp.PARAMETER1_PRM_SET/SAV  
+     recvBuffComp.PARAMETER2_PRM_SET/SAV
+
+     sendBuffComp.PARAMETER3_PRM_SET/SAV  
+     sendBuffComp.PARAMETER4_PRM_SET/SAV
+
+    """
+    
+    
+    # 1st remove file PrmDB.dat (manually)
+    # setup directory to remove later
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Svc.FileManager') + '.' + 'CreateDirectory', ["/tmp/PrmDb"], max_delay=5)    
+    
+    # send PRM_SAVE_FILE (Wrote 0 records)
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Svc.PrmDb') + '.' + 'PRM_SAVE_FILE', max_delay=1)
+    # Verify PrmDB.dat is empty    open /tmp/save_file1 (PrmDb.dat file = 0bytes )
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Svc.FileManager') + '.' + 'ShellCommand', ["ls -lagt", "/tmp/PrmDb/save_file1"], max_delay=5)
+
+
+    ## setup default-value
+    fprime_test_api.send_and_assert_command(fprime_test_api.getCmdDispName('Ref.RecvBuff') + '.' + 'PARAMETER1_PRM_SET', [5], max_delay=5)
```

Comment:
The component InT tests should not use Ref-specific components like `Ref.RecvBuff`. Unfortunately, there isn't a deployment-agnostic way to fully test PrmDb. I think this whole test should be moved to the Ref/test/int script.

This test script would only test a nominal `PRM_SAVE_FILE`, which will unfortunately write 0 records.